### DISCUSSION TITLE
Capacity credit

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#118] [Q&A: Analysis]
Hi ReEDS techs, When I read the capacity credit section of the 2021 ReEDS technical report, I was confused by two terms. 

"The resulting existing and marginal capacity credit values then feed into ReEDS to quantify each VRE resource’s capacity contribution to the planning reserve requirement. Existing VRE capacity credit calculations are performed by region and technology. For all candidate VRE resources that might be built in the coming year, the marginal capacity credit is calculated by region, technology, and resource class."

What is the difference between existing and marginal capacity credit in ReEDS? Thanks!

**Comment by vcelloho** [comment]
It's similar to the difference between an average and marginal value. The existing VRE capacity credit measures the total amount of capacity that a given VRE contributes towards the planning reserve. These are calculated using the Augur module and are based upon the installed capacity and how it would be utilized.

ReEDS when selecting investments needs to understand what additional capacity value a candidate VRE would have. Augur calculates the marginal capacity credit for all candidate VRE, this representing what fraction of the next unit of installed VRE specific to region, technology, and class, that should be expected.

As an example when we discuss the marginal capacity credit of PV going to zero in high VRE scenarios, this applies specifically to new installations. The existing PV does provide firm capacity, new installations won't increase this value.

**Comment by wanyingw193** [comment]
Hi, in 2021 ReEDS technical report, there are Capacity Credits (CC) and Capacity Values (CV). What is the difference between these two terms? Attached is a screenshot.

1
![image](https://user-images.githubusercontent.com/78582696/170780789-75df9929-d980-4fc6-a5e5-c94da356558c.png)
2
![image](https://user-images.githubusercontent.com/78582696/170780663-9f158ad2-29b7-45ee-8d57-7a77e80b0345.png)
3
![image](https://user-images.githubusercontent.com/78582696/170780532-01d2dd76-9397-4626-9f48-bb487079307e.png)

**Reply by vcelloho** [reply]
This is a case where we are mixing terms the two should be considered interchangeable.

**Comment by wanyingw193** [comment]
Hi! In Frew et al. (2017) _https://www.nrel.gov/docs/fy17osti/68869.pdf_, 100 hrs and 100 MW are used as proxies for capacity credit calculations using LDC-based approach. Why are 10 hours and 1000 MW suitable as default proxies for solar and wind capacity credit calculations in ReEDS? Why was 40 hours not chosen as a proxy, since H17 represents the highest hours of peak demand? Can the 1000 MW value be changed based on user input, just like the 10 hour proxy can be changed in value_defaults.csv? Thank you!

**Reply by wesleyjcole** [reply]
The current method in ReEDS uses the top 10 hours per season, so there are 40 hours total that are considered.  We chose the top 10 hours using a variety of internal interations and comparisons with the PRAS model the does ELCC estimates.  Some of that is documented in the supplemental information of https://www.sciencedirect.com/science/article/pii/S0306261920312782 where we shifted from using a single year of load and resource data to seven years of load and resource data (see Figures 19 and 20 in the SI).

The 1000 MW value was chosen by looking at typical build rates in our scenarios.  We found 1000 MW to be more representative than 100 MW.

The 1000 MW value is in cases.csv (marg_vre_mw) and can easily be adjust there.

**Reply by wanyingw193** [reply]
Thank you!
